testdata/golang/transpile/sort/implementation/IsSorted.java

```diff
@@ -1,6 +1,6 @@
 package com.eval;
 
-class Sort {
+class IsSorted {
```

Comment:
You need to rename the package directory as well since that is what the prompt prints as package path...